content/cloud/data-redis.md

```diff
@@ -59,32 +59,24 @@ The template we are interested in is `http-rust`.
 
 ## Creating Our New Spin Application
 
-The official Spin CLI documentation also has instructions on how to [create a new Spin application](/cloud/cli-reference#new). Let's go ahead and create a new `http-rust` application: 
+The official Spin CLI documentation also has instructions on how to [create a new Spin application](/cloud/cli-reference#new). Let's go ahead and create a new `http-rust` application (use up/down arrow keys to select `http-rust` when prompted):
```

Comment:
Any reason to ask the reader to do this rather than giving `spin new -t http-rust whatever`?  I would think that the concrete instruction gave the reader fewer places they could come adrift.